<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chat_Frame</class>
 <widget class="QWidget" name="Chat_Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_chat_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-30</y>
     <width>551</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resorces/resorces/login.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="display_box">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>425</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
  </widget>
  <widget class="QFontComboBox" name="fontComboBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>213</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="font_size">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>330</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
  </widget>
  <widget class="QToolButton" name="button_qqface">
   <property name="geometry">
    <rect>
     <x>278</x>
     <y>330</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resorces/qqface/51.gif</normaloff>:/resorces/qqface/51.gif</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="button_font_color">
   <property name="geometry">
    <rect>
     <x>307</x>
     <y>330</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resorces/image/textcolor.png</normaloff>:/resorces/image/textcolor.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="button_send_file">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>330</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resorces/image/fileopen.png</normaloff>:/resorces/image/fileopen.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="button_chat_history">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>330</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>聊天记录</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_qq_show">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>80</y>
     <width>128</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resorces/image/QQ_decorate.PNG);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="dialog_box">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>350</y>
     <width>425</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_close_chat">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>435</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_send_msg">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>435</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_friend_portrait">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>54</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/resorces/resorces/1.bmp</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_friend_name">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>54</width>
     <height>11</height>
    </rect>
   </property>
   <property name="text">
    <string>好友名</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
